src/util/health-check.sh

```diff
@@ -14,12 +14,13 @@
 
 TIMEOUT=10
 
-read -t ${TIMEOUT} -r input
+# prefer read with timeout (bash, busybox sh) or fall back to POSIX read (dash)
+read -t ${TIMEOUT} -r INPUT 2>/dev/null || read -r INPUT
 
-exit_code=$?
-cleaned_input=${input//[^a-zA-Z0-9]/}
+EXIT_CODE=$?
+CLEANED_INPUT=$(echo ${INPUT} | sed "s/[^a-zA-Z0-9]//g")
```

Comment:
Actually i would prefer lowercasing all variables as the uppercase ones usually are thought of as environment variables. 

I think i did TIMEOUT in upper case to show that it's a constant but it should be lowercase too.

Reply:
I have updated the "variable style" commit to standardize them on lower case.